interface-definitions/firewall.xml.in

```diff
@@ -115,6 +115,79 @@
               #include <include/generic-description.xml.i>
             </children>
           </tagNode>
+          <node name="dynamic-group">
+            <properties>
+              <help>Firewall dynamic group</help>
+            </properties>
+            <children>
+              <tagNode name="address-group">
+                <properties>
+                  <help>Firewall dynamic address-group</help>
+                  <constraint>
```

Comment:
I'm not sure if we should use hyphenated `address-group` here or just write "address group" without a hyphen here and throughout, where it's just a noun and not a keyword.